src/elasticsearch/elasticsearch.module.ts

```
@@ -1,17 +1,58 @@
 // src/elasticsearch/elasticsearch.module.ts
 import { Module } from '@nestjs/common';
-import { ElasticsearchService } from './elasticsearch.service';
+import { TypeOrmModule } from '@nestjs/typeorm';
```

ESLint

[error] Replace `'@nestjs/typeorm'` with `"@nestjs/typeorm"`

(prettier/prettier)